apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -26,7 +26,8 @@ export default function SchoolZipSearch({fieldNames, zip, disabled}) {
 
   useEffect(() => {
     if (!disabled) {
-      const searchUrl = `/dashboardapi/v1/schoolsearch/${zip}/40`;
+      setSelectedSchoolNcesId(SELECT_A_SCHOOL);
```

Comment:
I realized that if I typed in a zip code and then erased it and typed a new zip, the selected school would still remain in the dropdown even though it was no longer in the dropdown options. This clears that out between searches to avoid that scenario!

Reply:
Maybe curious if it's better to set this to null or an empty string. but this is turning into a nit so not blocking :)

Reply:
That all happens on the back end- it gets cleared out to an empty string if the school isn't found/selected! This is to reset that dropdown to the 'select a school' value